lib/Test2/API.pm

```diff
@@ -1300,7 +1300,7 @@ formatter being used.
 In both cases events are generated and stored in an array. This array is
 eventually used to populate the C<subevents> attribute on the
 L<Test2::Event::Subtest> event that is generated at the end of the subtest.
-This flag has no effect on this part, it always happens.
+This flag has no effect on this part: it always happens.
```

Comment:
I think the above should be a semicolon, not a colon.

Reply:
okay, changed.